Source/Particles/MultiParticleContainer.cpp

```diff
@@ -388,11 +388,26 @@ MultiParticleContainer::ReadParameters ()
 WarpXParticleContainer&
 MultiParticleContainer::GetParticleContainerFromName (const std::string& name) const
 {
```

Comment:
This generalizes the function `GetParticleContainerFromName` so that it also works with laser particles